src/OAuth/OAuth2/Service/Imgur.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace OAuth\OAuth2\Service;
+
+use OAuth\OAuth2\Token\StdOAuth2Token;
+use OAuth\Common\Http\Exception\TokenResponseException;
+use OAuth\Common\Http\Uri\Uri;
+use OAuth\Common\Consumer\CredentialsInterface;
+use OAuth\Common\Http\Client\ClientInterface;
+use OAuth\Common\Storage\TokenStorageInterface;
+use OAuth\Common\Http\Uri\UriInterface;
+
+class Imgur extends AbstractService
+{
+    /**
+     * Defined scopes
+     *
+     * @link http://www.reddit.com/dev/api/oauth
+     */
+    // Unable to find any scopes in the docs, but I could be blind.
+    //const SCOPE_FUBAR                        = 'foo';
```

Comment:
If you didn't find any then don't include anything here, including the commenting above.